cbrain_task/civet_macaque/bourreau/civet_macaque.rb

```diff
@@ -456,7 +459,7 @@ def save_results
     unless File.directory?("#{out_dsid}/logs")
       self.addlog("Error: this CIVET run did not complete successfully.")
       self.addlog("We couldn't find the log subdirectory '#{out_dsid}/logs' !")
-      return false # Failed On Cluster
+      saved_to_class = FailedCivetOutput # Failed On Cluster
```

Comment:
... and so here that would change to

```ruby
  is_partial = true
```

Reply:
Also, remove the comment `# Failed On Cluster`, it was only meaningful when we were returning 'false'.